pkg/operator/ceph/csi/csi.go

```diff
@@ -330,5 +330,33 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 		CSIParam.EnableVolumeGroupSnapshot = false
 	}
 
+	kubeApiBurst := k8sutil.GetValue(r.opConfig.Parameters, "CSI_KUBE_API_BURST", "")
+	if err != nil {
+		logger.Warningf("failed to load CSI_KUBE_API_BURST. %v", err)
+	}
+	CSIParam.KubeApiBurst = defaultKubeApiBurst
```

Comment:
Let's not have defaults here because we need to keep track of it and change it if something changes in the kubernetes sidecar containers, Provide an option to set these values, if they are set by admin/user then set that in csi driver deployment